src/software/sensor_fusion/sensor_fusion.cpp

```diff
@@ -5,7 +5,8 @@
 #include "software/parameter/dynamic_parameters.h"
 
 SensorFusion::SensorFusion()
-    : field(std::nullopt),
+    : history_size(20),
```

Comment:
We. should probably pull this into a const somewhere?

Reply:
yes it should but that should be done at the same time as Robot and Ball, which will be refactored during this project, so I want to hold off on it